transcom/mymove
# INT-B-20618 Office users edit/create/delete mobile home shipments

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20618)

## Summary

Most of this ticket was actually rendered OBE by Aaron's work on B-20617 (his PR is here: #13642).

However while looking into this story I found that feature flag checks were missing on the frontend for the "Add a new shipment" dropdown that is at the top of the "Move Details" page when an office user clicks on a move. There was also a flag check missing from the Prime UI "Create Shipment" form.

So, this branch adds checks for both of these, if you edit the .envrc to disallow mobile home shipments, you should no longer see an option to choose mobile home shipment types from either of these menus.

### How to test

1. Edit the .envrc and make the mobile home flag set to FALSE (running a find for "mobile" will take you to the flag).
2. Access a move as an SC.
3. Click the "Add a new shipment" dropdown and confirm that "Mobile Home" is **not** an option.
4. Repeat step 3 for TIO/TOO.
5. Access a move as a Prime sim user.
6. Click the "Create Shipment" button on the page, then on the next form confirm that "MOBILE_HOME" is **not** an option for the shipment type.
